docs_user/modules/proc_adopting-telemetry-services.adoc

```diff
@@ -14,93 +14,128 @@ This guide also assumes that:
+          storage:
+            strategy: persistent
+            retention: 24h
+            persistent:
+              pvcStorageRequest: 20G
+'
+----
+
+* Verify that `alertmanager` and `prometheus` pods are available
++
+----
+oc get pods -l alertmanager=metric-storage -n openstack
+NAME                            READY   STATUS    RESTARTS   AGE
+alertmanager-metric-storage-0   2/2     Running   0          46s
+alertmanager-metric-storage-1   2/2     Running   0          46s
+
+oc get pods -l prometheus=metric-storage -n openstack
+NAME                          READY   STATUS    RESTARTS   AGE
+prometheus-metric-storage-0   3/3     Running   0          46s
+----
 
-. Patch the `OpenStackControlPlane` CR to deploy Ceilometer services:
+.Procedure
+
+* Patch the `OpenStackControlPlane` CR to deploy Ceilometer services:
 +
 ----
-oc patch openstackcontrolplane openstack --type=merge --patch-file ceilometer_patch.yaml
+oc patch openstackcontrolplane openstack --type=merge --patch '
+spec:
+  telemetry:
+    enabled: true
+    template:
+      ceilometer:
+        enabled: true
+        secret: osp-secret
+'
 ----
 
 .Verification
 
 . Inspect the resulting Ceilometer pods:
 +
 ----
-CEILOMETETR_POD=`oc get pods -l service=ceilometer | tail -n 1 | cut -f 1 -d' '`
+CEILOMETETR_POD=`oc get pods -l service=ceilometer -n openstack | tail -n 1 | cut -f 1 -d' '`
 oc exec -t $CEILOMETETR_POD -c ceilometer-central-agent -- cat /etc/ceilometer/ceilometer.conf
 ----
 
-. Inspect the resulting Ceilometer IPMI agent pod on data plane nodes:
+. Inspect enabled pollsters:
 +
 ----
-podman ps | grep ceilometer-ipmi
+oc get secret ceilometer-config-data -o jsonpath="{.data['polling\.yaml\.j2']}"  | base64 -d
 ----
 
-. Inspect enabled pollsters:
+. Optional: Override default pollsters according to requirements:
 +
 ----
-oc get secret ceilometer-config-data -o jsonpath="{.data['polling\.yaml']}"  | base64 -d
+oc patch openstackcontrolplane controlplane --type=merge --patch '
+spec:
+  telemetry:
+    template:
+      ceilometer:
+          defaultConfigOverwrite:
+            polling.yaml.j2: |
+              ---
+              sources:
+                - name: pollsters
+                  interval: 100
+                  meters:
+                    - volume.*
+                    - image.size
+          enabled: true
+          secret: osp-secret
+'
 ----
 
-. Enable pollsters according to requirements:
+* Patch the `OpenStackControlPlane` CR to include `logging`
 +
 ----
-cat << EOF > polling.yaml
----
-sources:
-    - name: pollsters
-      interval: 300
-      meters:
-        - volume.size
-        - image.size
-        - cpu
-        - memory
-EOF
-
-oc patch secret ceilometer-config-data  --patch="{\"data\": { \"polling.yaml\": \"$(base64 -w0 polling.yaml)\"}}"
+oc patch openstackcontrolplane openstack --type=merge --patch '
+spec:
+  telemetry:
+    template:
+      logging:
+      enabled: false
+      network: internalapi
```

Comment:
Just realized now due to an bug that this field does not exist anymore. It needs to be removed from the guide.